content/docs/intro.md

```diff
@@ -12,13 +12,13 @@ Start with getting familiar with the [geodesic](/reference/tools.mdx#geodesic).
 
 Get intimately familiar with docker inheritance and [multi-stage docker builds](/reference/best-practices/docker-best-practices.md#multi-stage-builds). We use this pattern extensively.
 
-Check out our [terraform-aws-components](https://github.com/cloudposse/terraform-aws-components) for reference architectures to easily provision infrastructure
+Check out our [terraform-aws-components](https://github.com/cloudposse/terraform-aws-components) for reference architectures to easily provision infrastructure.
 
 ## Tools
 
-Tons of tools/clis are used as part of our solution. We distribute these tools in a couple of different ways.
+Tons of tools/CLIs are used as part of our solution. We distribute these tools in a couple of different ways:
 
-* Geodesic bundles most of these tools as part of the geodesic base image
+* Geodesic bundles most of these tools as part of the geodesic base image:
```

Comment:
Doesn't seem like there should be a `:` here as this is an item in a list.

Reply:
You are right! I missed it. Everything else seems fine, should I be closing the pull request now?

Reply:
Let's remove this colon and merge. The rest of the diff make sense to me.